src/app/index.scss

```diff
@@ -828,6 +830,10 @@ option[label^="Removed"] {
         text-align: center;
     }
 
+    &.vertical-center {
```

Comment:
Would it make more sense to call this something like "center-crosswise"? If the flex-axis of the container is vertical, then this would be a "horizontal-center", which wouldn't make sense

Reply:
Or maybe see if the existing "center" class could be used to just "center everything". If that doesn't break other stuff it might simplify things until we do need to center in only one axis. I'm not sure we need to yet

Reply:
This is used to place the "X" (delete button) on the developer status.  It moves the "X" down so it aligns correctly with the other `flex-item`s in the "row", since the other `flex-item`s are taller than the one with the "X".

Reply:
So a "general center" wouldn't work, but the name of the class could be something specific to the fact that it's centering across the main axis, as opposed to centering vertically. It's only centering vertically if the main axis is horizontal, and we wouldn't want to need to create a new class called "horizontal-center" with the same attributes just to do the same work on the different axis